### ISSUE TITLE
Latest Version of Connector / Extras / Forgified = ParticleRain suddenly asking/crashing about 'clothconfig2' (nvm bigger issue inside i guess)

### ISSUE THREAD

**Issue by hiimrickross** [Sinytra/Connector#1082]
### Mod Name

Particle Rain

### Mod Homepage

https://www.curseforge.com/minecraft/mc-mods/particle-rain

### Minecraft version

1.20.1

### Describe the bug

Compared to beta 36, extras 1.9.2, and forgified 0.92.0.1.11.3, suddenly loading MC is now prompting this error. Im not sure if its because I've updated these mods to their recent/most recent versions: Amethyst Imbuement, PuffishSKills, Simply Skills, Wizards/Archers/Paladins, Gearifiers (and their dependencies). Previous I'd had very old versions of these mods, along with connector 36, and it seemed to be ok. Though I did have a similar crash once that I resolved by adjusting some mods.. So im scratching my head here cus I can't quite remember the situation. 

### Steps to reproduce

1. Install latest cloth config (forge)
2. Install Particle Rain
3. Install latest connector/extras/etc
...
Error on startup requesting cloth config2. Even tho connector aliases has cloth config 2 covered already.

### Logs

https://gist.github.com/hiimrickross/422ae56c4f919fc92d5b52fea2cad22d

**Comment by hiimrickross** [comment]
https://gist.github.com/hiimrickross/00e7b4bc5d5c3f3f72b9b6620f08db2d

https://github.com/Sinytra/Connector/discussions/67 i tried to follow this also to make a dependency override, but now rather than crash requsting cloth config it just exit code 1's me and i can't find anything in the log

**Comment by hiimrickross** [comment]
OK its something to do with the connector updates and some other mod interaction I think

I reverted removed all of the mods that interact with each other; runes, spell power/engine, (RPG series mod), gearifier, etc, simplyskills and puffishskills, even quark & zeta which i recently updated just in case it was those.

Then I reverted back to fabric api 0.92.0-1.11.3, connector beta 36, connector extras 1.9.2. Cleared the connector folder, did the first boot into normal crash, then second boot loaded up into menu fine.

cleared connector files and replaced with the latest, first boot normal crash yep, but second boot crashed again, even with all those mods gone.

https://gist.github.com/hiimrickross/e9ba73de36a644d6d6265447631af3c3

**Comment by liangmoe** [comment]
All the global aliases should migrate to connector.json sine beta-37, read https://sinytra.org/docs/connector/configuration

**Comment by Tapacywka** [comment]
`Trying to update SS 1.3.2 -> requires newer spell engines and spell powers -> These will load after crashing the first time. But no skill trees are being generated (unknown if this is their issue or connector issue, the combination worked just fine on SS 1.2.1's setup.`

> In the recent update the mod was split, the core part is [Pufferfish's Skills](https://www.curseforge.com/minecraft/mc-mods/puffish-skills). Attributes are a separate mod [Pufferfish's Attributes](https://www.curseforge.com/minecraft/mc-mods/puffish-attributes). The default skill trees were also removed, but can still be downloaded as a data pack [Default Skill Trees](https://www.curseforge.com/minecraft/data-packs/default-skill-trees).

**Comment by Tapacywka** [comment]
Newest spell engine / spell power / putterfish skills / simple skills / all RPG Series mod / Particle Rain working good for me with the latest Connector / Extras / FFAPI / Lazurite / Embeddium / Oculus.

**Comment by hiimrickross** [comment]
I resolved the Simply Skills Issue; versions 1.3.0+ have a hard incompatibility under the hood with Prominence Classic modpack where it intentionally doesn't work fully. Probably works fine outside of that modpack, but maybe there's some beef idk/idc, I talked to the dev and found that out. Stuck on 1.2.1 when it comes to Prominence.

The problem with Particle Rain more than likely stemmed from another mod incompatibility that just made that a red herring instead, haven't checked and it's been a while but I recall it being the case long ago. Will try add PR back later.

However the main issue remains - I cannot update v36 Connector to latest v43 (and dependencies), the game will crash exit code 1 on launch every single time. Error logs tell me nothing obvious, so I don't know what's crashing and why v36 works completely fine (and I've updated from v27~ to v36 over time and never had issues) but going to this fails now.

I've got the primary log: https://gist.github.com/hiimrickross/d893eac1007f2bc0594ed169c1e587af , and the debug log: https://gist.github.com/hiimrickross/1e25f75e74a5e86d7408319918b46b52

**Comment by hiimrickross** [comment]
Holy CRAP I finally tracked down the issue.

I was trying to trouble shoot something completely different - an oddity where setting resource packs would work when loading the game, and getting into a world, etc, all plays fine! However resource pack reload fail would happen if trying to set when in a world, or after exiting back to main menu, would only work again after full close and open.

A lucky crash which hadn't happened before while reloading, pointed to Mythic Upgrades ... Which I removed, and suddenly both Connector V43 loads up and into the world, and the resource pack reload fail problem doesn't happen any more either!

I HOPE this helps somebody... It was very bizarre that Connector 36-> 41/42 worked, but 43 would not work. Luckily through troubleshooting something else, I solved that... Now I am on fully latest connector/extras/forgified.
